Source/WebCore/dom/WindowEventLoop.h

```diff
@@ -82,6 +88,10 @@ class WindowEventLoop final : public EventLoop {
 
     std::unique_ptr<CustomElementQueue> m_customElementQueue;
     bool m_processingBackupElementQueue { false };
+
+    bool m_hasARenderingOpportunity { false };
+    MonotonicTime m_lastIdlePeriodStartTime;
+    MonotonicTime m_nextRenderingUpdateTime;
```

Comment:
What is the purpose of these two times?

Reply:
Oh oops, these are useless in this version. Will remove.